src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,77 @@
+package codeu.controller;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import java.io.PrintWriter;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class AdminServlet extends HttpServlet {
+
+	//Store clas that gives access to Users. 
+	private UserStore userStore;
+
+	// set state for request on server
+	public void init() throws ServletException {
+		super.init();
+		setUserStore(UserStore.getInstance());
+	}
+	
+	// set UserStore used by this servlet
+	void setUserStore(UserStore userStore) {
+		this.userStore = userStore;
+	}
+
+	
+	@Override
+	//this is to check for the users 
+
+	public void doGet (HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException{
+		request.getRequestDispatcher("/login").forward(request, response);
+
+	}
+
+ 	@Override
+        //this checks to see if the user is one of the admins
+        public void doPost (HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException{
```

Comment:
What is the user submitting here? Usually you only need `doPost()` if you have a form that the user is entering data into. Do you have a form on the admin page or something?

Taking a step back, I'd try to work forward in smaller steps. Get the `doGet()` part working before you try to get the `doPost()` part working.

I'm not sure if you've seen this already, but take a look at the first video in the video examples page under the project section of the website. That's an example of adding a servlet and a JSP file without any extra stuff and is a decent example of how I'd approach it.

Also take a look at Ana's prototype, this is a great example of working forward in small steps: https://github.com/costaa17/CodeU-Spring-2018/pull/5